src/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -53,14 +46,16 @@ const Wrapper = styled.div`
   }
 `
 
-const Container = styled.div<{ sticky: boolean }>`
+const Container = styled.div<{ sticky: boolean; hasDiscount: boolean }>`
+  position: ${(props) => (props.sticky ? `fixed` : `relative`)};
+  ${(props) => props.sticky && `top: 6rem`};
   width: 26rem;
-  background-color: ${colorsV2.white};
+  padding: 2rem 0 1.5rem 0;
```

Comment:
```suggestion
  padding: 2rem 0 1.5rem;
```

Reply:
i don't agree with this actually - i think its harder to read 😬 because i never remember which direction the 3rd measure isin